test/controllers/site_controller_test.rb

```diff
@@ -1,7 +1,22 @@
 require 'test_helper'
 
 class SiteControllerTest < ActionController::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  
+   
+  test "should get index" do
+    get :index
```

Comment:
It seems like this tests is not really necessary SiteControllerTest#index is basically displaying a static page, this test is testing the Rails framework, not our application. What do you think?